Assignment-1-6/form.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+include __DIR__ . '/emailValidate.php';
+
+// Default value.
+$valid_phone = TRUE;
+$email_check = '';
+
+// If Form submitted.
+if (isset($_POST['submit'])) {
+  // Get first_name & last_name.
+  $first_name = $_POST['first-name'];
+  $last_name = $_POST['last-name'];
+
+  // Get image.
+  if (isset($_FILES['image']) && $_FILES['image']['error'] == UPLOAD_ERR_OK) {
+    $target_file = './images/' . basename($_FILES['image']['name']);
+
+    // Move uploaded image to 'images' folder.
+    if (move_uploaded_file($_FILES['image']['tmp_name'], $target_file)) {
+      $img_upload = 'successful';
+    }
+    else {
+      $img_upload = 'unsuccessful';
+    }
+  }
+
+  // Get sujects and marks string.
+  $marks_str = $_POST['subject-marks'];
+  // Convert the string in array of `sub|mark'.
+  $marks_arr = explode("\n", $marks_str);
+  // Convert the $marks_arr array in associative array of `sub => mark`.
```

Comment:
Just explaining the business logic is fine.
We don't need to explain every line of code. :)